pycbc/waveform/compress.py

```diff
@@ -423,7 +430,7 @@ def compress_waveform(htilde, sample_points, tolerance, interpolation,
     'double': numpy.float64
 }
 
-def fd_decompress(amp, phase, sample_frequencies, out=None, df=None,
+def fd_decompress(amp, phase, amp_freq, phase_freq, out=None, df=None,
```

Comment:
This isn't what we agreed to in #1083. Only the frequency points for the phase should be required. Amplitude frequency points should be a keyword argument, with the default set to None. So, have a required argument called `sample_frequencies` that is (assumed to be) both for phase and amplitude, and a keyword `amp_sample_frequencies` that, if specified, will be used for the amplitude, with the other only being used for the phase.